### ISSUE TITLE
Using with react-native-navigation

### ISSUE THREAD

**Issue by moneymc** [lucasferreira/react-native-flash-message#34]
When i use it with RNN it's looks like this:
![image](https://user-images.githubusercontent.com/3620646/55251895-96b18600-5262-11e9-9246-2d01169f2768.png)

How i implement:

```
const ProviderWrap = Comp => props => (
  <View>
    <Comp {...props} />
    <FlashMessage position="top" hideStatusBar={true} />
  </View>
);
Navigation.registerComponent("Login", () => ProviderWrap(Login));
```

**Comment by lucasferreira** [comment]
Hi @moneymc,

Could you follow the solution exposed in this other issues?

https://github.com/lucasferreira/react-native-flash-message/issues/13
https://github.com/lucasferreira/react-native-flash-message/issues/21
https://github.com/lucasferreira/react-native-flash-message/issues/18

TL;DR: You have to put de _<FlashMessage />_ "above" de navigators, like this demo here: https://snack.expo.io/Hy7Y01Yc7

**Comment by tyleralves** [comment]
All of your links reference `react-navigation` which is pretty different from `react-native-navigation` despite the names.

**Comment by lucasferreira** [comment]
Oh I'm sorry @tyleralves,

I didn't get it at first, but now I see that you using other lib instead.

I've never used this lib before (`react-native-navigation`) but as I can see this libs works in native layer of RN _(Obj-C or Java)_, in a area where my JavaScript FlashMessage lib can't interfere, so at first I guess there the only way to used is to put the <FlashMessage /> component inside any of your views and work locally (instead of globally).

**Comment by MatthewPattell** [comment]
@tyleralves hi, just add:
```js
Navigation.showOverlay({
		component: {
			name:      'app.topBarFlashMessages',
			passProps: {},
			options:   {
				overlay: {
					interceptTouchOutside: false,
				},
				layout:  {
					backgroundColor: 'transparent',
					orientation:     ['portrait'],
				},
			},
		},
	})
```

and register component:

```js
Navigation.registerComponent('app.topBarFlashMessages', () => TopFlashMessages)
```

Component:

```js
const styles = StyleSheet.create({
	container: {
		height: 50,
		width:  wp('100%'),
	},
})

const TopFlashMessages= () => {
	return (
		<View style={styles.container}>
			<FlashMessage position="top" />
		</View>
	)
}
```

**Comment by quiringk** [comment]
FYI it wasn't working for me on the first time you called `showMessage` because I wasn't waiting for `showOverlay` to finish. When the `showOverlay` promise completes then call `showMessage`
